config/application.rb

```diff
@@ -49,6 +49,13 @@ class Application < Rails::Application
       end
     end
 
+    config.to_prepare do
```

Comment:
I would've thought that this is equivalent to the above:

```ruby
    config.to_prepare do
      # Allows us to use decorator files in the app directory
      Dir.glob(Rails.root.join("app/**/*_decorator*.rb")).sort.each do |c|
        Rails.configuration.cache_classes ? require(c) : load(c)
      end

      Dir.glob(Rails.root.join("lib/**/*_decorator*.rb")).sort.each do |c|
        Rails.configuration.cache_classes ? require(c) : load(c)
      end
    end
```

Just curious if that's not the case.

Reply:
I am grasping at straws in this PR.